doc/source/architecture.rst

```diff
@@ -3,17 +3,15 @@
 Architecture
 ********************************************************************************
 
-|pbm| uses one |pbm-agent| process per |mongod| node. The :term:`PBM Control
-collections` in the |mongodb| cluster or non-sharded replica set itself serve as
-the central configuration, authentication and coordination
-channel. Administrators observe and control the backups or restores with a
-|pbm.app| CLI command that they can run from any host with the access to the
-|mongodb| cluster.
-
-A single |pbm-agent| is only involved with one cluster (or non-sharded replica
-set). The |pbm.app| CLI tool can connect to any cluster it has network access
-to, so it is possible for one user to list and launch backups or restores on
-many clusters. 
+|pbm| consists of the following components:
+
+- |pbm-agent| is a process running on every ``mongod`` node within the cluster or a replica set that performs backup and restore operations. 
+- **pbm** CLI is a command-line utility that instructs pbm-agents to perform an operation. 
+
+  A single |pbm-agent| is only involved with one cluster (or non-sharded replica set). The |pbm.app| CLI utility can connect to any cluster it has network access to, so it is possible for one user to list and launch backups or restores on many clusters. 
+
+- :term:`PBM Control collections` is a special collection in MongoDB that stores the configuration data and backup states. Both |pbm.app| CLI and |pbm-agent| use PBM Control collections to authenticate in MongoDB and communicate with each other. 
```

Comment:
```suggestion
- :term:`PBM Control collections` are special collections in MongoDB that stores the configuration data and backup states. Both |pbm.app| CLI and |pbm-agent| use PBM Control collections to check backup status in MongoDB and communicate with each other. 
```
The PBM control collections aren't part of the authentication. The MongoDB Authentication system is used just as it already is.